src/api.js

```diff
@@ -7,6 +7,13 @@ export const getArticles = () => {
     })
 }
 
+export const getArticle = (id) => {
+
+    return axios.get(`https://backend-nc-project-w9aw.onrender.com/api/articles/${id}`).then(({data}) => {
+        return data
+    })
+}
+
```

Comment:
Have a look at the following notes on a nicer way to do this:

l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance